release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3322,6 +3322,41 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+/ 4.14.29
+[id="ocp-4-14-29"]
+=== RHBA-2024:3697 - {product-title} 4.14.29 bug fix update and security update
+
+Issued: 2024-06-12
+
+{product-title} release 4.14.29, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2024:3697[RHBA-2024:3697] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:3700[RHSA-2024:3700] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.14.29 --pullspecs
+----
+
+[id="ocp-4-14-29-bug-fixes"]
+==== Bug fixes
+
+* Previously, for {product-title} deployments on {rh-openstack-first}, the `MachineSet` object did not correctly apply the value for the `Port Security` parameter. This meant that the `port_security_enabled` parameter in the {rh-openstack} server's port would have an unexpected value. With this release, the `MachineSet` object applies the `port_security_enabled` flag as expected. (link:https://issues.redhat.com/browse/OCPBUGS-32428[*OCPBUGS-32428*])
+
+* Previously, when an `IngressController` was configured with client SSL/TLS, but did not have the `clientca-configmap` finalizer, the Ingress Operator would try to add the finalizer without checking whether the IngressController was marked for deletion. Consequently, if an `IngressController` was configured with client SSL/TLS and was subsequently deleted, the Operator would correctly remove the finalizer. The Operator would then repeatedly, and erroneously, try and fail to update the `IngressController` to add the finalizer back, resulting in error messages in the Operator's logs. With this update, the Ingress Operator no longer adds the `clientca-configmap` finalizer to an `IngressController` that is marked for deletion. As a result, the Ingress Operator no longer tries to perform erroneous updates, and no longer logs the associated errors. (link:https://issues.redhat.com/browse/OCPBUGS-34410[*OCPBUGS-34410*])
```

Comment:
```suggestion
* Previously, when an `IngressController` was configured with client SSL/TLS, but did not have the `clientca-configmap` finalizer, the Ingress Operator would try to add the finalizer without checking whether the IngressController was marked for deletion. Consequently, if an `IngressController` was configured with client SSL/TLS and was subsequently deleted, the Operator would correctly remove the finalizer. The Operator would then repeatedly, and erroneously, try and fail to update the `IngressController` to add the finalizer back, resulting in error messages in the Operator's logs. With this update, the Ingress Operator no longer adds the `clientca-configmap` finalizer to an `IngressController` that is marked for deletion. As a result, the Ingress Operator no longer tries to perform erroneous updates and no longer logs the associated errors. (link:https://issues.redhat.com/browse/OCPBUGS-34410[*OCPBUGS-34410*])
```

Reply:
+1. 

/remove-label peer-review-needed
/remove-label peer-review-in-progress
/label peer-review-done